flink-connectors/flink-connector-cassandra/src/test/java/org/apache/flink/streaming/connectors/cassandra/CassandraConnectorITCase.java

```diff
@@ -438,6 +461,27 @@ public void cancel() {
 	}
 
 	@Test
+	public void testCassandraTableSink() throws Exception {
+		StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
+		env.setParallelism(1);
+
+		DataStreamSource<org.apache.flink.types.Row> source = env.fromCollection(rowCollection);
+		CassandraTableSink cassandraTableSink = new CassandraTableSink(new ClusterBuilder() {
+			@Override
+			protected Cluster buildCluster(Cluster.Builder builder) {
+				return builder.addContactPointsWithPorts(new InetSocketAddress(HOST, PORT)).build();
+			}
+		}, INSERT_DATA_QUERY, new Properties());
+		cassandraTableSink.configure(FIELD_NAMES, FIELD_TYPES);
```

Comment:
Yes. It would also be good to test the TableSink in an actual Table API program. Most of the methods like `configure()` and `emitDataStream()` are internally called by the API.